wallet/src/repository/wallet/tests/mod.rs

```diff
@@ -1101,3 +1111,64 @@ fn test_create_vtt_with_locked_balance() {
         err,
     );
 }
+
+#[test]
+fn test_create_vtt_with_multiple_outputs() {
```

Comment:
Is there a test to check for non-well serialized addresses?

Reply:
I added tests in the `create_vtt` validation logic. :)